docs/releasing.rst

```diff
@@ -0,0 +1,34 @@
+:orphan:
```

Comment:
Should we not link this from a toctree?

Reply:
There's no toctree. It's using an implicit one that covers all the sections of the main document.